cookbooks/fb_users/README.md

```diff
@@ -170,6 +171,27 @@ automatically cleaned up.
 
 Also see `initialize_group` helper below.
 
+The `notifies` is a hash that looks like this:
+
+```ruby
+node.default['fb_users']['users']['zerocool'] = {
+  'restart foo if uid changes' => {
+    'resource' => 'service[foo]',
+    'action' => 'restart',
+  }
```

Comment:
this should probably have a timer option too, specifically since the default timer is `:delayed` and most of the use cases I can think of require `:immediately`
```suggestion
    'timer' => :immediately,
  }
```

Reply:
I added it, but I called it 'timing' to not be confused wiuth systemd timers.